package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "alanlauer.com",
+  "version": "1.0.0",
```

Comment:
lets mark the site version as prior to version 1 for now. we can use 0.0.1 until we have a consensus on how we will handle this field